packages/typespec-azure-resource-manager/lib/models.tsp

```diff
@@ -348,10 +348,9 @@ model ResourceListResult<Resource extends Foundations.SimpleResource> is Azure.C
  */
 @doc("The custom response of a list operation.")
 @friendlyName("{name}ListResult", Result)
-@pagedResult
```

Comment:
The removal of `@pagedResult` on the model would likely means break and require immediate action from TypeSpec source to adapt to `@list` -- some of the migration uses `ResourceActionSync` to specify a List API (i.e., existing code in specs repo, and all the migration PR that not switched to `@list` on API, would not work with the new version of typespec-azure-resource-manager lib)

Please let us know whether this is expected.
Or we prefer to keep this `@pagedResult` for a month or two, before a final switch.

Reply:
Yeah, I can get that breaks come sooner or later. I just thought there would be a few months buffer (that pending PR can still use the old pattern).

Unmerged PR to specs repo won't get fixed automatically.

SDK repo has tons of [commitID](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storageactions/azure-resourcemanager-storageactions/tsp-location.yaml#L2) that need to be updated upon next release (and typically the TypeSpec source on HEAD has other difference from the pinned commitID).
And there are unmerged tsp-location files in pending PR.

All of them need to be handled upon the break. I just need to know when it happens... I don't have strong objection if "when" is "next release".

Reply:
we brought this up a month and a half ago and talked about it, originally the plan was to have this last release and was postponed to next one(early august).

Now as said before this is not urgent and if adding `@pageItems` with `@azure.core.items` really helps out smooth the migration effort we can just do that now and do the breaking part in september

Reply:
Sure, let me circle it around in SDK devs. If no one have strong opinion, let's say we can just go break right now.

Reply:
Its quite a simple change so i have a pr there too if we want to got that way first https://github.com/Azure/typespec-azure/pull/3028

Reply:
Thanks Timothee. So far, there is no outstanding feedback from dev asking for buffer time. I think we can go directly in.